src/v3/test/integration/authenticator-verification-email.test.tsx

```diff
@@ -124,13 +125,20 @@ describe('authenticator-verification-email', () => {
         },
       );
 
+      act(() => {
+        jest.advanceTimersByTime(31_000);
+      });
+      await findByText(/Haven't received an email?/);
       // render invalid otp message
       const codeEle = await findByTestId('credentials.passcode') as HTMLInputElement;
       const submitButton = await findByText('Verify', { selector: 'button' });
       const verificationCode = '123456';
       await user.type(codeEle, verificationCode);
       await user.click(submitButton);
       await findByText('Invalid code. Try again.');
+      // After an error, verify that the Reminder prompt is removed in lieu of the global error
+      expect(queryByText(/Haven't received an email?/)).toBeNull();
+      await findByText(/We found some errors./);
```

Comment:
Can you also update this test file to test that the reminder prompt re-appears again given enough time after the error?